spark/arctern_spark/geodataframe.py

```diff
@@ -30,6 +30,55 @@
 
 class GeoDataFrame(DataFrame):
     def __init__(self, data=None, index=None, columns=None, dtype=None, copy=False, geometries=None, crs=None):
+        """
+        A GeoDataFrame object is a pandas.DataFrame that has columns with geometry.
```

Comment:
A GeoDataFrame object is a pandas.DataFrame that has columns with **geometries.**